### DISCUSSION TITLE
Raspberry Pi with eth0 and wlan0 bridged, and assigns IP addresses to eth0 and wlan0 clients

### DISCUSSION THREAD

**Discussion by daltzctr** [RaspAP/raspap-webgui#1417] [General]
I effectively want to accomplish the following configuration.

![Untitled Diagram drawio(1)](https://github.com/RaspAP/raspap-webgui/assets/105223895/e289b8b0-881e-442e-852a-1de482b37e39)

I tried the bridged AP docs over on the raspberry pi documentation site, but they skip out on the DHCP server. I believe there are shenanigans that must be done to assign IPs to bridged clients, but I'm quite a rookie in this regard. 

Any assistant is appreciated. Thanks!

**Comment by billz** [comment]
Ethernet bridging isn’t possible with WiFi interfaces in STA mode — only in AP. This is a limitation of the 802.11 spec.
Routing between these interfaces can be done, but not with RaspAP. There are a million posts on the RPi forums about this.

Related #1403

**Reply by daltzctr** [reply]
I did end up rolling my own configuration that bridged eth0 and wlan0, then assigned addresses to the bridge

**Reply by athshean** [reply]
How did you do this? I've been trying to do the same but have had no luck.
